niworkflows/anat/ants.py

```diff
@@ -298,10 +305,15 @@ def init_brain_extraction_wf(name='brain_extraction_wf',
         (dil_brainmask, get_brainmask, [('output_image', 'op1')]),
         (inu_n4_final, apply_mask, [('output_image', 'in_file')]),
         (get_brainmask, apply_mask, [('output_image', 'mask_file')]),
-        (get_brainmask, outputnode, [('output_image', 'out_mask')]),
-        (apply_mask, outputnode, [('out_file', 'out_file')]),
-        (inu_n4_final, outputnode, [('output_image', 'bias_corrected'),
+        (get_brainmask, copy_xform, [('output_image', 'out_mask')]),
+        (apply_mask, copy_xform, [('out_file', 'out_file')]),
```

Comment:
You have `(apply_mask, copy_xform, [('out_file', 'out_file')]),` here and `(copy_xform, apply_mask, [('out_mask', 'mask_file')]),` below. I'm pretty sure this is going to break.